README.md

```diff
@@ -283,7 +283,10 @@ And finally, you are ready to go! Or at least you should be...
 
 ### Junior Contributors
 
+**Lucas Silva** | [brachingo] | I love driving, I have an Opel Corsa Sport
+and a Kawasaki, and driving them makes me feel free
 
+[brachingo]: <https://github.com/brachingo>
```

Comment:
Almost!

You must place it inside of the link list becouse where it is at this moment it will get deleted due your mates names. Just place it inside the list(sorted alphabetically) or being the last.